composer.json

```diff
@@ -10,11 +10,16 @@
         {"name": "Steve Müller", "email": "st.mueller@dzh-online.de"}
     ],
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.3",
+        "squizlabs/php_codesniffer": "~1.0"
```

Comment:
Ok I noticed this change because in doctrine/DoctrineModule composer was broken :)